src/lighteval/tasks/lighteval_task.py

```diff
@@ -190,6 +190,16 @@ def __init__(  # noqa: C901
 
         current_categories = [metric.category for metric in self.metrics]
         self.has_metric_category = {category: (category in current_categories) for category in MetricCategory}
+        if (
```

Comment:
we do not need this, as we can use transformers for llm as judge

Reply:
Hm this feels like it was ported back with a merge or stg, removing indeed